src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -44,10 +44,10 @@ protected override void Initialize()
         {
             base.Initialize();
 
-            ForegroundThreadAffinitizedObject.CurrentForegroundThreadData = ForegroundThreadData.CreateDefault();
+            ForegroundThreadAffinitizedObject.CurrentForegroundThreadData = ForegroundThreadData.CreateDefault(defaultKind: CommandLineMode);
```

Comment:
I know I told you to call it `CommandLineMode`, but since we don't actually test that it is command line mode, I don't like that anymore :)

Maybe `ForcedByPackageInitialize`?